src/api/spec/controllers/webui/request_controller/show_spec.rb

```diff
@@ -0,0 +1,360 @@
+require 'rails_helper'
+
+RSpec.describe Webui::RequestController, '#show', vcr: true do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package_with_file, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, :as_submission_source, name: 'ball', project: source_project) }
+  let(:bs_request) do
+    create(:bs_request_with_submit_action,
+           description: 'Please take this',
+           creator: submitter,
+           target_package: target_package,
+           source_package: source_package)
+  end
+
+  describe 'GET #show' do
+    context 'when the user is part of the beta program' do
+      render_views
+
+      before do
+        Flipper.enable(:request_show_redesign, receiver)
+      end
+
+      context 'when the user has only one submit action' do
+        let!(:comment) { create(:comment, commentable: bs_request) }
+
+        before do
+          login receiver
+          get :show, params: { number: bs_request.number }
+        end
+
+        it 'shows the new redesign page' do
+          expect(response.body).to have_text("Request #{bs_request.number}")
+          expect(response.body).to have_text('beta program')
```

Comment:
I'm gonna remove those template checking from all the spec, because that is testing a specific implementation and doing that makes our tests brittle. We should avoid doing that. But I need to check that those templates are used indirectly... I marked the PR as draft, because I'm changing stuff and it's not really "for review" right now...